jsonld/credit-classes/C01-verified-carbon-standard-class.json

```
@@ -0,0 +1,73 @@
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "regen": "http://regen.network/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "dash": "http://datashapes.org/dash#",
+    "schema:itemListElement": { "@container": "@list" },
+    "regen:sectoralScope": { "@container": "@list" },
+    "regen:offsetGenerationMethod": { "@container": "@list" }
+  },
+  "@type": "regen:CreditClass",
+  "schema:description": "",
+  "schema:name": "",
+  "schema:url": {
+    "@type": "schema:URL",
+    "@value": ""
+  },
+  "regen:sourceRegistry": {
+    "schema:name": "",
+    "schema:url": {
+      "@type": "schema:URL",
+      "@value": ""
+    }
+  },
+  "regen:sectoralScope": [
+    ""
```

@wgwz do we want to add a type template here so folks know what should be put in? 

```suggestion
    {
      "@type": "xsd:string",
      "@value": ""
    }
```

---

So at some point @S4mmyb I thought we needed to have that explicit typing for the `xsd:string` but @blushi clarified that in this case we don't need to so explicit. Honestly, I'm not 100% clear on the reasons why the explicit typing seemed to be required for certain cases in order for validation to happen. I know we had some of those. I attribute it to me just  eing a noob with RDF ;-)

In any case, I think Marie wants this to just be an empty string. So I think it's OK as is.

---

I should have said, that so far, in these JSONLD templates, when we have an example that's a string, we seem to be favoring just showing an empty string, rather than some example/fake value. so `""` over `"Foo Inc."` So it's more that we're just trying to be consistent with that (I guess..)

---

Also, I guess, that the way you've specified the string data:

```
[
    {
      "@type": "xsd:string",
      "@value": ""
    }
]
```

Is perfectly equivalent to `[""]`. It's two different ways to represent this, I think it means exactly the same thing in RDF perspective.

---

Marking resolved :-)

---

awesome, thanks!

---

On https://regen-registry-server.herokuapp.com/metadata-graph/regen:13toVgqN4ESXgLNQZNz7iGPNuwr7HghWxNCk4W5JrsaX2NwXfNJVg9K.rdf, the C01 credit class metadata was still using xsd:string format, I've updated it to just use strings so we keep things consistent (the generated IRI is unchanged since the data is exactly the same as @wgwz pointed out).

---

FYI @S4mmyb I've updated the credit class metadata to just use strings for the CO1 class in the db

---

sure! thanks